src/Components/PlayPauseButton/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { PlayCircle } from "styled-icons/boxicons-regular/PlayCircle";
+import { PauseCircle } from "styled-icons/boxicons-regular/PauseCircle";
+
+const Button = styled.button`
+  color: ${props => (props.color ? props.color : "#000")};
+  background-color: Transparent;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+`;
+
+const PlayPauseButton = ({ isPlay = true, color, size = 48, ...props }) => {
```

Comment:
We shouldn't default boolean props to true. The reason is because boolean props are usually just not passed in to a component to signify that we want it to be false (or undefined).

- [ ] remove default value from `isPlay`